src/lib/SnackBar/missing-types.d.ts

```diff
@@ -0,0 +1,13 @@
+import { SnackOptions } from '.';
+
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      'snack-bar': SnackBarAttributes;
+    }
+
+    interface SnackBarAttributes extends HTMLAttributes {
+      showSnackbar?: (options: SnackOptions) => void;
```

Comment:
This isn't an attribute, is it? I don't think it shows up on the JSX at all right?